packages/common/src/adapters/track.ts

```diff
@@ -219,3 +222,119 @@ export const stemTrackMetadataFromSDK = (
     is_owned_by_user: false
   }
 }
+
+export const trackMetadataForUploadToSdk = (
+  input: TrackMetadataForUpload
+): TrackMetadata => ({
+  ...camelcaseKeys(
+    pick(input, [
+      'title',
+      'description',
+      'license',
+      'isrc',
+      'iswc',
+      'genre',
+      'mood',
+      'tags',
+      'is_unlisted',
+      'field_visibility',
+      'is_premium',
+      'premium_conditions',
+      'is_stream_gated',
+      'stream_conditions',
+      'is_download_gated',
+      'download_conditions',
+      'orig_file_cid',
+      'orig_filename',
+      'is_downloadable',
+      'is_original_available',
+      'bpm',
+      'is_custom_bpm',
+      'musical_key',
+      'is_custom_musical_key',
+      'comments_disabled',
+      'ddex_release_ids',
+      'parental_warning_type',
+      'audio_upload_id',
+      'ai_attribution_user_id'
+    ])
+  ),
+  title: input.title,
```

Comment:
There are a number of fields in this section that get assigned by the `camelcaseKeys` call above and then reassigned. Was that intentional?

Reply:
No, I forgot to remove them when I added custom logic for specific fields